1.19.3-pre3 stuff

Shnupbups opened the pull request.

Shnupbups committed:
1.19.3-pre3 stuff

YanisBft reviewed (approved):
Mind including https://github.com/FabricMC/yarn/commit/100acb44f5158eb477ebe546f62005528d208b4f#r91439942 in your PR so it's not forgotten?

Shnupbups committed:
javadoc fix

https://github.com/FabricMC/yarn/commit/100acb44f5158eb477ebe546f62005528d208b4f#r91439942

enbrain commented on mappings/net/minecraft/sound/SoundEvent.mapping:
```diff
@@ -15,3 +17,11 @@ CLASS net/minecraft/class_3414 net/minecraft/sound/SoundEvent
+	METHOD method_47958 writeBuf (Lnet/minecraft/class_2540;)V
+		ARG 1 buf
+	METHOD method_47959 getStaticDistanceToTravel ()Ljava/util/Optional;
+	METHOD method_47961 readBuf (Lnet/minecraft/class_2540;)Lnet/minecraft/class_3414;
```
```suggestion
	METHOD method_47961 fromBuf (Lnet/minecraft/class_2540;)Lnet/minecraft/class_3414;
```
CONVENTIONS.md says
> Use "read" for non-static methods that load data into the object.
> Use "from" for static methods that create an object of the method owner's type (`fromJson`, `fromNbt`, `fromString`).

apple502j committed:
Update mappings/net/minecraft/sound/SoundEvent.mapping

Co-authored-by: enbrain <69905075+enbrain@users.noreply.github.com>
